turbinia/notify.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Sends email notifications'''
+
+import logging
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from turbinia import config
+log = logging.getLogger('turbinia')
+
+#Sends notfications via email
+def sendmail(subject, message):
+  '''Sends an Email notification via SMTP'''
+  try:
+    if config.EMAIL_NOTIFICATIONS is True:
+
+      #Puts message in MIME format
+      msg = MIMEMultipart()
+      msg['From'] = config.EMAIL_ADDRESS
+      msg['To'] = config.EMAIL_RECIEVING_ADDRESS
+      msg['Subject'] = subject
+      msg.attach(MIMEText(message))
+
+      #start SMTP
+      server = smtplib.SMTP(config.EMAIL_HOST_ADDRESS, config.EMAIL_PORT)
+
+      # identify to mail server
+      server.ehlo()
+
+      # secure email using TLS
+      server.starttls()
+
+      # reidentify as using TLS
+      server.ehlo()
+
+      #Attempt to login
+      try:
+        server.login(config.EMAIL_ADDRESS, config.EMAIL_PASSWORD)
```

Comment:
Is there any reason not to check if these exist first and just skip the log in if they are empty?  I haven't really used the NameError exception before, but it looks to me like that is used when the variable itself doesn't exist, but I believe these will exist in the config even if they are set to None.